capitalize_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class CapitalizeDecorator < Nameable
+  def initialize(nameable = nameable.new)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Here `CapitalizeDecorator`should  Inherits from Decorator. && is no need to initialize again. Kindly remove the initialized part.
 ```ruby
class CapitalizeDecorator < Decorator
```